text_extract.py

```diff
@@ -0,0 +1,29 @@
+import os
+import re
+from glob import glob
+import hwp5
+# from util import makedir
+
+class HWPExtractor():
+	def __init__(self):
+		# try:
+		# 	import hwp5
+		# except:
+		# 	print('Package "pyhwp" is requred to extract HWP files.')
+		# 	print('pip install --pre pyhwp')
+		pass
+
+
+	def convert(self, hwp_file, output_file):

```

Comment:
개인적으로 아래와 같이 명명하면 좋을 것 같음.

file: 파일 객체(object)
fname: 파일 이름
fdir: 파일이 속한 폴더 경로
fpath: 파일의 전체 경로(=os.path.join(fdir, fname))

Reply:
코멘트 반영 및 documentation 추가. 아래 코드도 이에 따라 수정함.
```suggestion
	def convert(self, hwp_fpath, output_fpath):
	'''
	hwp_fpath: path of hwp file to be converted
	output_fpath: path of output (txt) file containing raw text
	'''

```